public/app/features/alerting/unified/components/rule-viewer/tabs/Details.tsx

```diff
@@ -50,6 +50,8 @@ const Details = ({ rule }: DetailsProps) => {
     ? rule.annotations ?? []
     : undefined;
 
+  const hasEvaluationDuration = Number.isFinite(evaluationDuration);
```

Comment:
can be infinite? 🤔

Reply:
No, but if it's `0` (which is a valid number we accept) it would be `false` even though it _is_ a valid duration :)